runtime/src/lib.rs

```diff
@@ -82,19 +101,19 @@ pub mod opaque {
 
     /// Opaque block type.
     pub type Block = generic::Block<Header, UncheckedExtrinsic>;
+    pub type BlockId = generic::BlockId<Block>;
+}
 
-    pub type SessionHandlers = ();
-
-    impl_opaque_keys! {
-        pub struct SessionKeys {}
-    }
+#[cfg(not(feature = "standalone"))]
+impl_opaque_keys! {
+    pub struct SessionKeys {}
 }
 
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("VLN"),
-    impl_name: create_runtime_str!("vln-parachain"),
+    impl_name: create_runtime_str!("vln-runtime"),
     authoring_version: 1,
-    spec_version: 100,
+    spec_version: 1,
```

Comment:
I remember reading in some issue somewhere that as convention the parachains where going to use the spec version starting at 100, we can check

Reply:
Fixed in https://github.com/valibre-org/vln-node/pull/69/commits/b3c3956df5232948ca31e9e6e6c69c5d2326e6ec